index.bs

```diff
@@ -517,7 +530,8 @@ spec: webidl
       The <dfn>"camera"</dfn>, <dfn>"microphone"</dfn> , and
       <dfn>"speaker"</dfn>
       permissions are associated with permission to use media devices as
-      specified in [[GETUSERMEDIA]] and [[audio-output]].
+      specified in [[GETUSERMEDIA]] and [[audio-output]]. {{"speaker"}} is
```

Comment:
@alvestrand Did I get the right set of media-related permissions that require secure contexts?